ci-operator/config/openshift/backplane-cli/openshift-backplane-cli-main.yaml

```
@@ -28,6 +28,11 @@ tests:
     make test
   container:
     from: src
+- as: scan
+  commands: |
+    make scan
+  container:
+    from: src
```

This should be an optional check

```suggestion
- as: scan
  commands: make scan
  container:
    from: src
  optional: true
```